source/main.cpp

```diff
@@ -68,9 +69,12 @@ int main(int argc, char *argv[])
 	if(argc > 1)
 		InitConsole();
 #endif
+	Test::Context testContext;
```

Comment:
Does this need to have the same scope as the variable that says which test is run?

Reply:
No, didn't need to. Moved one scope smaller.